README.md

```diff
@@ -59,19 +59,55 @@ with sycl_context("gpu"):
 daal4py API, allows you to use wider set of Intel(R) oneAPI Data Analytics Library algorithms in just one line:
 ```py
 import daal4py as d4p
-d4p.kmeans_init(data, 10, t_method="plusPlusDense")
+init = d4p.kmeans_init(data, 10, t_method="plusPlusDense")
+result = init.compute(X)
 ```
-You can even run this on a cluster by simple adding a keyword-parameter
+You can even run this on a cluster by simple code changes:
 ```py
 import daal4py as d4p
+d4p.daalinit()
 d4p.kmeans_init(data, 10, t_method="plusPlusDense", distributed=True)
-```
+result = init.compute(X, daal4py.my_procid())
+d4p.daalfini()
+```
+
+# Scikit-learn patching
+
+| *Speedups of daal4py powered Scikit-learn over the original Scikit-learn, 28 cores, 1 thread/core* |
+|:--:|
+| ![](doc/IDP%20scikit-learn%20accelearation%20compared%20with%20stock%20scikit-learn.png) |
+| *technical details: FPType: float32; HW: Intel(R) Xeon(R) Platinum 8276L CPU @ 2.20GHz, 2 sockets, 28 cores per socket; SW: scikit-learn 0.22.2, Intel® DAAL (2019.5), Intel® Distribution Of Python (IDP) 3.7.4; Details available in the article https://medium.com/intel-analytics-software/accelerate-your-scikit-learn-applications-a06cacf44912* |
+
+daal4py patching will affect performance for specific Scikit-learn functionality listed below. In some cases daal4py can fallback into stock Scikit-learn, if there are some unsupported parameters (these limitations described below). If the patching doesn't cover your scenarios - please, submit an issue on GitHub.
```

Comment:
Need to add information about the verbose mode before description about Sklearn parameters.